unix/src/machine_stats/plugins/cpu_utilization.py

```diff
@@ -0,0 +1,27 @@
+def setup(app):
+    timeout = app.args.cpu_utilization_timeout
+    if timeout > 0:
+        app.add_playbook_tasks(
+            dict(
+                action=dict(module="cpu_utilization", args=dict(timeout=timeout)),
+            )
+        )
+
+
+def add_arguments(parser):
+    parser.add_argument(
+        "--cpu-utilization-timeout",
+        metavar="SECONDS",
+        type=int,
+        default=30,
+        help="timeout sampling for CPU utilization",
+    )
+
+
+def ok_callback(parent, result):
+    host = result._host.get_name()
+    cpu_util = result._result.get("ansible_cpu_utilization")
+    if cpu_util is not None:
+        parent.update_results(
+            host, {"cpu_average": cpu_util["average"], "cpu_peak": cpu_util["peak"]}
```

Comment:
This is a non breaking change to the existing machine-stats workflow. Did I get that correct?

Reply:
The final output is the same as previous one. Two more fields are added to the final output: `cpu_average` and `cpu_peak`.